### ISSUE TITLE
[BUG] transform-none not detected

### ISSUE THREAD

**Issue by thebuilder** [francoismassart/eslint-plugin-tailwindcss#84]
**Describe the bug**
After upgrading to Tailwind 3, the eslint plugin gives the warning:

```
Classname 'transform-none' is not a Tailwind CSS class
```

Might be related to `transform` itself being removed?

**To Reproduce**

1. Add the `transform-none` class to an element
2. Run the eslint task

Adding utility classes causes the same warning, e.g. `motion-reduce:transform-none` (which would still make sense if you need to disable animations)

https://tailwindcss.com/docs/translate#removing-transforms

**Comment by francoismassart** [comment]
FIXED in `3.0.3`, I also added a commented test in order to detect a contradicting `transform-none`

https://github.com/francoismassart/eslint-plugin-tailwindcss/commit/e2a0bb49c2344e376baca6b0fdce62b49a887e0b#diff-21e03577c146459846b540d5af5f11c98b829e063384e946627c94f9f834db9cR391-R397

**Comment by francoismassart** [comment]
Thank you for reporting the issue @thebuilder 👍

**Comment by thebuilder** [comment]
Thanks for the quick fix!
